community-operators/kubevirt/kubevirtoperator.package.yaml

```diff
@@ -0,0 +1,4 @@
+packageName: kubevirt
```

Comment:
Why do we have two packages?

Reply:
Two packages because one for upstream-community-operator that run's on K8s and community- operator for OKD/Openshift.

Reply:
Ah. Thanks